app/src/main/java/com/lorenzovainigli/foodexpirationdates/view/composable/screen/SettingsScreen.kt

```diff
@@ -251,4 +296,16 @@ fun SettingsScreenContentPreview() {
             SettingsScreen()
         }
     }
-}
\ No newline at end of file
+}
+
+fun setScreenProtectionEnabled(context: Context, enabled: Boolean) {
+    val sharedPreferences = context.getSharedPreferences("my_preferences", Context.MODE_PRIVATE)
+    val editor = sharedPreferences.edit()
+    editor.putBoolean("screen_protection_enabled", enabled)
+    editor.apply()
+}
+
+fun getScreenProtectionEnabled(context: Context): Boolean {
+    val sharedPreferences = context.getSharedPreferences("my_preferences", Context.MODE_PRIVATE)
+    return sharedPreferences.getBoolean("screen_protection_enabled", false)
+}
```

Comment:
Can you move these functions into [PreferencesRepository.kt](https://github.com/lorenzovngl/FoodExpirationDates/blob/main/app/src/main/java/com/lorenzovainigli/foodexpirationdates/model/repository/PreferencesRepository.kt) to keep consistency?
Also please try to use variables like this one below to avoid string repetitions:

https://github.com/lorenzovngl/FoodExpirationDates/blob/6bac4bf2e9573f150496d57faaffc0cdf9ee3ad1/app/src/main/java/com/lorenzovainigli/foodexpirationdates/model/repository/PreferencesRepository.kt#L16